### ISSUE TITLE
Relink 2.3.7 seems to have issues with escaping parenthesis in markdown tiddlers

### ISSUE THREAD

**Issue by Klemet** [flibbles/tw5-relink#50]
Hello there 😄 ! And thank you for the Relink plugin, it is wonderful ! 

I think I discovered a little issue remaining. It seems that markdown tiddlers with parenthesis in the titles become "missing" according to the `More/Relink (Missing)` list; and if their title is changed, links in other tiddlers are not updated. This problem does not appear with non-markdown "normal" tiddlers.

![image](https://github.com/flibbles/tw5-relink/assets/41070328/f9d12d75-3504-4788-954a-10c062df9e31)

(the `Relink (Missing)` list on the screenshot is very long because I have a lot of tiddlers with parenthesis in their title 😬).

I've installed the Relink markdown sub-plugin, and it has been working perfectly for any other markdown link, as long as the tiddler in the link doesn't have a parenthesis in its title.

I imagine that the problem might be in the way that the parenthesis characters are escaped ?

**Comment by flibbles** [comment]
It probably is about the parenthesis escaping. It'll take a hot minute before I have the free time to look into this, but I will when I can.

**Comment by Klemet** [comment]
No worries; thank you for considering it, it's very appreciated !

**Comment by flibbles** [comment]
All right. I might actually need more context for this. I'm not seeing this problem on my end.

Can I see the raw markdown of how you're making your links? This is how I am making mine:
```
[Markdown Test Tiddler With (parenthesis)](#Markdown%20Test%20Tiddler%20with%20(parenthesis))
```

**Comment by Klemet** [comment]
Sure ! It seems that when I do a link in a markdown tiddler to another markdown tiddler that has parenthesis in its name using `Ctrl+L`, it escapes the characters.

![image](https://github.com/flibbles/tw5-relink/assets/41070328/de1ba3c0-3872-4e49-8125-8f48cbecf66e)

So it seems that in my file, the title of a markdown file with parenthesis is dealt internally by escaping the parenthesis.

**Comment by flibbles** [comment]
Issue should be fixed with Relink-Markdown release v2.3.8.

**Comment by Klemet** [comment]
Just installed it, and it works like a charm, many thanks 😄 !! I'm amazed by how quickly it was fixed and updated !
